homeassistant/components/zwave/__init__.py

```diff
@@ -126,8 +126,8 @@
 SET_NODE_VALUE_SCHEMA = vol.Schema(
     {
         vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
-        vol.Required(const.ATTR_VALUE_ID): vol.Coerce(int),
-        vol.Required(const.ATTR_CONFIG_VALUE): vol.Coerce(int),
+        vol.Required(const.ATTR_VALUE_ID): vol.Any(vol.Coerce(int), cv.string),
+        vol.Required(const.ATTR_CONFIG_VALUE): vol.Any(vol.Coerce(int), cv.string),
```

Comment:
Please update [services.yaml](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/zwave/services.yaml) as well to reflect these made changes:
https://github.com/home-assistant/home-assistant/blob/41f3fb291afe6596f276400e8c692e815172f40d/homeassistant/components/zwave/services.yaml#L77-L80

Reply:
Updated.